applications/StructuralMechanicsApplication/custom_conditions/base_load_condition.cpp

```diff
@@ -366,6 +366,38 @@ void BaseLoadCondition::AddExplicitContribution(
 /***********************************************************************************/
 /***********************************************************************************/
 
+const Parameters BaseLoadCondition::GetSpecifications() const
+{
+    const Parameters specifications = Parameters(R"({
+        "time_integration"           : ["static","implicit","explicit"],
+        "framework"                  : "lagrangian",
+        "symmetric_lhs"              : true,
+        "positive_definite_lhs"      : true,
+        "output"                     : {
+            "gauss_point"            : [],
+            "nodal_historical"       : [],
+            "nodal_non_historical"   : [],
+            "entity"                 : []
+        },
+        "required_variables"         : ["DISPLACEMENT"],
+        "required_dofs"              : ["DISPLACEMENT_X","DISPLACEMENT_Y","DISPLACEMENT_Z"],
+        "flags_used"                 : [],
+        "compatible_geometries"      : ["Line2D2", "Line2D3", "Triangle3D3", "Triangle3D6", "Quadrilateral3D4", "Quadrilateral3D8", "Quadrilateral3D9"],
+        "is_implicit"                : true,
```

Comment:
It is redundant as it is given in the "time_integration" field.

Reply:
I just follow the requirements from the @KratosMultiphysics/technical-committee, if the @KratosMultiphysics/technical-committee agrees I will remove it

Reply:
@KratosMultiphysics/technical-committee agrees on removing it and check the `is_implicit `by `[time_integration].Has("implicit")`